[2019-05-16 04:05:11,227] - LogUtil.py] - DEBUG: 666
